doc/source/getting_started.rst

```diff
@@ -0,0 +1,388 @@
+    arr = LArray(data, [age, sex], "population by age and sex")
+    arr
+
+Axes can be also provided as a single with each axis separated by a semi-colon:
+
+.. ipython:: python
+   :doctest:
+
+    arr = LArray(data, "age=0..2;sex=M,F", "population by age and sex")
+    arr
+
+Here are the key properties for an array:
+
+.. ipython:: python
+   :doctest:
+
+    # array summary : dimensions + description of axes
+    arr.info
+
+    # number of dimensions
+    arr.ndim
+
+    # array dimensions
+    arr.shape
+
+    # number of elements
+    arr.size
+
+    # size in memory
+    arr.memory_used
+
+    # type of the data of the array
+    arr.dtype
+
+Special arrays can be generated through dedicated functions:
+
+* :py:func:`ndrange` : fills an array with increasing numbers
+* :py:func:`ndtest` : same as ndrange but with axes generated automatically (for testing)
+* :py:func:`empty` : creates an array but leaves its allocated memory unchanged (i.e., it contains “garbage”. Be careful !)
+* :py:func:`zeros` : fills an array with 0
+* :py:func:`ones` : fills an array with 1
+* :py:func:`eye` : identity matrix
+* :py:func:`full` : fills an array with a given value
+
+Save/Load an array
+------------------
+
+The LArray library offers many I/O functions to read and write arrays in various formats
+(CSV, Excel, HDF5, pickle). For example, to save an array in a HDF5 file, call the method
+:py:meth:`~LArray.to_hdf`:
+
+.. ipython:: python
+   :doctest:
+
+    arr.to_hdf('my_file.h5', 'arr')
+
+To load a saved array, call the function :py:meth:`read_hdf`:
+
+.. ipython:: python
+   :doctest:
+
+    arr = read_hdf('my_file.h5', 'arr')
+    arr
+
+Other functions are described :ref:`here <api-IO>`.
+
+Indexing
+--------
+
+You can select an element or a subset of an array by labels and/or positions.
+
+Using labels
+^^^^^^^^^^^^
+
+To select an element or a subset of an array using labels, use brackets [ ].
+Let’s start by selecting a single element:
+
+.. ipython:: python
+   :doctest:
+
+    arr = ndtest((4, 4))
+    arr
+
+    arr['a0', 'b1']
+
+    # labels can be given in an arbitrary order
+    arr['b1', 'a0']
+
+To select a row or column, specify the associated label:
+
+.. ipython:: python
+   :doctest:
+
+    # first row
+    arr['a0']
+
+    # first column
+    arr['b0']
+
+To take a subset, you can use a list of labels or a slice (defined by 'start:stop:step').
+Arguments `start`, `stop` and `step` of a slice are optional. By default, `start` represents
+the first label of an axis, `stop` the last one. Argument `step` is 1 by default.
+
+.. ipython:: python
+   :doctest:
+
+    # 'b1:b3' is a shortcut for 'b1,b2,b3'
+    arr['a0,a2', 'b1:b3']
+
+    # ':a2' will select all labels between the first one and 'a2'
+    # 'b1:' will select all labels between 'b1' and the last one
+    arr[':a2', 'b1:']
+
+    # 'a0::2' is a shorcut for 'a0,a2'
+    # 'b3::-1' is a shortcut for 'b3,b2,b1,b0'
+    arr['a0::2', 'b3::-1']
```

Comment:
My fear was that some new users will read the getting started, scrap the tutorial and will use LArray for months without knowing that they can provide a step. But it is their responsibility after all.